docs/environment_variables.md

```diff
@@ -132,6 +132,18 @@ Example:
 SSL_CERT_DIR=/path/to/ca-certs/ python -c "import httpx; httpx.get('https://example.com')"
 ```
 
+## `.NETRCFILE`
```

Comment:
I think this env name should be `NETRC`, right?

https://github.com/encode/httpx/blob/3231211aa364c0406737b78a45268e08c434f94c/httpx/_utils.py#L134-L135

and per the doc:
https://www.python-httpx.org/advanced/#netrc-support

Reply:
Yep! you're right, this should be 'NETRC', especially in the example part. I would like to confirm one thing that I noticed in the doc  (https://www.python-httpx.org/advanced/#netrc-support) both the right content table and central content title are written with a dot as the start, like ".netrc", so should I need to change this title to be ".NETRC" or simply "NETRC"?

Reply:
> I think this env name should be `NETRC`, right?
> 
> https://github.com/encode/httpx/blob/3231211aa364c0406737b78a45268e08c434f94c/httpx/_utils.py#L134-L135
> 
> and per the doc:
> https://www.python-httpx.org/advanced/#netrc-support

For consistency, '.NETRC' would be better👀may be.

Reply:
In this `Environment Variables` section, I think we should just use `NETRC`. 👍

Reply:
Gotcha! Thanks for this suggestion.